### PR TITLE
Fixes #122 changed the href of the link for participation guidlines and changed …

### LINKED ISSUE #122: Add link to Participation Guidelines in footer
https://github.com/mozilla/foundation.mozilla.org/issues/235
I'm under the impression that these changes have to take place here, so I'm going to reference this so I get some feedback from people here too.

### PR DESCRIPTION
**Pull request by jayyyin**
…the scss class to match

Issue #: https://github.com/mozilla/mofo-bootstrap/issues/122

NOTE: since the change of the scss class this is a breaking change and we need to do a version bump in package.json

### CODE REVIEW

**Comment by tiagomoraismorgado88** [comment]
looks better

**Comment by gideonthomas** [comment]
Thanks for the PR @jayyyin, I'll get someone from our team to review it.

File: src/index.pug [hunk 1]
```diff
@@ -236,7 +236,7 @@ html
           li.col-auto
             a(href="#" class="footer-link-cc-license") License
           li.col-auto
-            a(href="#" class="footer-link-code-of-conduct") Code of Conduct
+            a(href="https://www.mozilla.org/about/governance/policies/participation/" class="footer-link-participation-guidelines") Code of Conduct
```
Comment (gvn) [inline src/index.pug]: I'm unclear on why you're changing the class name. `footer-link-code-of-conduct` should still be accurate as it is a code of conduct and the link text is "Code of Conduct".

Comment (gideonthomas) [reply src/index.pug]: @jayyyin, the text should be Participation Guidelines

**Comment by jayyyin** [comment]
sorry it appears I missed the text when I changed it, must have miss-remembered changing the text for the other PR instead @gvn should be good now

**Comment by gideonthomas** [comment]
Thanks for your patience and work on this @jayyyin!

**Comment by jayyyin** [comment]
@gideonthomas np, thank you for following up =D

**Pull request merged**